embedchain/vectordb/vector_db.py

```diff
@@ -0,0 +1,159 @@
+from typing import Any, List, Optional, Union
+
+from langchain.docstore.document import Document
+
+from embedchain.vectordb.chroma_db import ChromaDB
+from embedchain.vectordb.elasticsearch_db import EsDB
+
+
+class VectorDb:

```

Comment:
Why this new class, why not use BaseVectorDb for this?

Reply:
agree, I have updated code with suggested changes.